master/reference/node/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment | Description                              | Schema | Default |
+| ------------- | -------- | ------ | ----- |
+| NODENAME | Node ID. | string | |
+| NO_DEFAULT_POOLS | Prevents calico from creating a default pool if one does not exist. | boolean | false |
+| HOSTNAME | Hostname. | string | |
+| IP | IP address which calico will bind to. | ip address | 10.0.0.1 |
+| IP6 | IPv6 address which calico will bind to. | ipv6 address | fed80::1 |
+| AS | AS number for this node. | int | global AS number |
+| DATASTORE_TYPE | Type of datastore. |can be `kubernetes` or `etcd` | etcd |
+| WAIT_FOR_DATASTORE | Prevent calico/node startup from initiating etcd connection until an etcd connection is made. | boolean | false |
+| CALICO_LIBNETWORK_ENABLED | Enables running the docker-libnetwork plugin directly in the calico/node container. | boolean | true |
+| CALICO_NETWORKING_BACKEND | Describes which BGP networking backend to use| can be `gobgp`, `bird`, `none` | bird |
+| **etcd configuration** | | | |
+| ETCD_ENDPOINTS    | A comma separated list of etcd endpoints (optional, defaults to http://127.0.0.1:2379) | http://etcd1:2379 | |
+| ETCD_KEY_FILE     | Path to the etcd key file (optional)       | /etc/calico/key.pem | |
+| ETCD_CERT_FILE    | Path to the etcd client cert (optional)    | /etc/calico/cert.pem | |
+| ETCD_CA_CERT_FILE | Path to the etcd CA file (optional)        | /etc/calico/ca.pem | |
```

Comment:
See: https://github.com/projectcalico/calico-containers/pull/1390 and https://github.com/projectcalico/calico/pull/334/files